kalkulator.py

```diff
@@ -5,12 +5,16 @@ def dodaj(a, b):
 wynik = a + b
 return wynik
 
+def odejmij(a, b):
+return a - b 
```

Comment:
Moim zdaniem tutaj lepiej rozbić to na dwie linijki wprowadzając zmienną "wynik".